dbfile.c

```diff
@@ -175,12 +175,36 @@ static int dbfile_set_modes(sqlite3 *db)
 {
 	int ret;
 
-	ret = sqlite3_exec(db, "PRAGMA synchronous = OFF", NULL, NULL, NULL);
+//	Setting synchronous off is unsafe
+//	It is much safer to just expand cache and the result is the same after warmup
+//	ret = sqlite3_exec(db, "PRAGMA synchronous = OFF", NULL, NULL, NULL);
+	ret = sqlite3_exec(db, "PRAGMA synchronous = 1", NULL, NULL, NULL);
 	if (ret) {
 		perror_sqlite(ret, "configuring database (sync pragma)");
 		return ret;
 	}
 
+//	Grow cache size see issue #283
+	ret = sqlite3_exec(db, "PRAGMA cach_size = -20000", NULL, NULL, NULL);
```

Comment:
`cache_size`?

Reply:
Ah yes my mistake.  That's what I get for just throwing something together using github as an editor.

Reply:
Fix in 3762657915c6de0ca063476ca09a4f22f75e350f but I'm not familiar with github's reviewing tools so not sure how be darned if I can figure out how to show that here.